apps/user/.env.example

```
@@ -1 +1,3 @@
 NEXT_PUBLIC_API_URL=http://localhost:3333/api/v1
+NEXT_PUBLIC_ADMIN_APP_URL=http://localhost:3001
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ADMIN_APP_URL key should go before the NEXT_PUBLIC_API_URL key

(UnorderedKey)